wire-grpc-tests/src/test/java/com/squareup/wire/MockRouteGuideService.kt

```diff
@@ -57,7 +57,7 @@ class MockRouteGuideService : RouteGuideGrpc.RouteGuideImplBase(), TestRule {
 
   suspend fun awaitSuccess() {
     try {
-      withTimeout(2000L) {
+      withTimeout(3000L) {
```

Comment:
Flaky tests seem to timeout here so adding a minute for the lucky strike, don't fire me

Reply:
yeah still don't understand